### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by kradical**
Some cleanup

### CODE REVIEW

File: meteor/forms.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
-from django import forms
+from django.contrib.gis import forms
+from django.contrib.auth.password_validation import validate_password
 
 
-class Createaccount(forms.Form):
+class CreateAccount(forms.Form):
```
Comment (ProgrammingJoe) [inline meteor/forms.py]: Good catch. I'll remember this for future class names.

File: meteor/views.py [hunk 2]
```diff
@@ -3,54 +3,73 @@
...
 
-
 def signin(request):
-    login_form = Login(request.POST or None)
-
     return render(
         request,
         'meteor/login.html',
-        {'login_form': login_form}
+        {'login_form': Login()}
```
Comment (ProgrammingJoe) [inline meteor/views.py]: Cool!

File: meteor/views.py [hunk 3]
```diff
@@ -3,54 +3,73 @@
...
+    try:
+        validate_email(email)
+    except ValidationError as e:
+        return JsonResponse({'error': 'invalid email ' + str(e)})
+    
+    if User.objects.filter(email=email):
+        return JsonResponse({'error': 'User already exists for that email'})
+        
+    new = User(first_name=first, last_name=last, password=password, organization=organization, email=email)
+    print(new)
```
Comment (ProgrammingJoe) [inline meteor/views.py]: Did you mean to keep this print? Also, do you want to merge my change and fix the conflict here?

Comment (kradical) [inline meteor/views.py]: Nope I thought I deleted it :laughing: will do!

**Pull request merged**